American Whitewater/Extensions/MKMapView+FitAnnotations.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  MKMapView+FitAnnotations.swift
+//  American Whitewater
+//
+//  Created by David Nelson on 11/20/19.
+//  Copyright © 2019 American Whitewater. All rights reserved.
+//
+
+import UIKit
+import MapKit
+import Foundation
+
+extension MKMapView {
```

Comment:
Just wondering, why are there separate files for `MKMapView+FitAll` and `MKMapView+FitAnnotations`? Seems like these are logically similar operations.